### ISSUE TITLE
Provide props to SortOn components of search block.

### ISSUE THREAD

**Issue by iFlameing** [plone/volto#4162]
Currently in sortOn component of Search block doesn't provide you to disable the searching capabilities of react Select.

**Comment by MAX-786** [comment]
Hello @iFlameing, I tried looking into this issue, I found the `Blocks/Search/components/SortOn.jsx`, but when I make changes inside the `SortOn()` function, they are not reflected in my project. 

I have not been able to figure out why the changes are not being reflected. Is there any way you could help me troubleshoot the issue or point me in the direction of a resource that might be able to help?
Thanks!

**Comment by wesleybl** [comment]
> when I make changes inside the SortOn() function, they are not reflected in my project.

When you say "my project", is it anything other than the Volto clone? I think the best way you can try to fix a problem is to clone the Volto and make changes to it.

**Comment by MAX-786** [comment]
> When you say "my project", is it anything other than the Volto clone?
No, it is the Volto clone 😅. Sorry "my project" was misleading.

**Comment by tiberiuichim** [comment]
@MAX-786 If you make changes in code that resides in `node_modules/@plone/volto`, you have to restart Volto (unless you have the latest released Volto, which includes hot-reloading for that source path).

You can also make a checkout of Volto, switch the branch and follow these instructions to "install" that volto branch in your project: https://github.com/plone/volto/issues/2810#issuecomment-970200321

**Comment by MAX-786** [comment]
> @MAX-786 If you make changes in code that resides in node_modules/@plone/volto, you have to restart Volto (unless you have the latest released Volto, which includes hot-reloading for that source path).

got my mistake. Thanks!
